app/models/export.rb

```diff
@@ -0,0 +1,10 @@
+class Export < ActiveRecord::Base
+  has_attached_file :file 
+  before_create :attach_file 
+
+  def attach_file 
+    users = User.all
+    csv = UsersExport(users).to_csv
+    # file = File.
+  end
```

Comment:
If this method is going to kick-off the csv generation, that I would name it as such.
Calling it `attach_file` seems opaque/overloaded